pkgs/by-name/xo/xonsh/xontribs/xonsh-direnv/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  direnv,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "xonsh-direnv";
+  version = "1.7.0";
```

Comment:
Instead of this bulk inclusion, why not including one by one?
This should not be so hard, and also will be helpful in future bisections.